Commands/InteractionCommands/ClearInteractions.cs

```diff
@@ -0,0 +1,264 @@
+﻿namespace Cliptok.Commands.InteractionCommands
+{
+    public class ClearInteractions : ApplicationCommandModule
+    {
+        public static Dictionary<ulong, List<DiscordMessage>> MessagesToClear = new();
+
+        [SlashCommand("clear", "Delete many messages from the current channel.", defaultPermission: false)]
+        [HomeServer, SlashRequireHomeserverPerm(ServerPermLevel.TrialModerator), RequireBotPermissions(Permissions.ManageMessages), SlashCommandPermissions(Permissions.ModerateMembers)]
+        public async Task ClearSlashCommand(InteractionContext ctx,
+            [Option("count", "The number of messages to delete. Required if you don't use the 'upto' argument.")] long count = 0,
+            [Option("up_to", "Optionally delete messages up to (not including) this one. Accepts IDs and links.")] string upTo = "",
+            [Option("user", "Optionally filter the deletion to a specific user.")] DiscordUser user = default,
+            [Option("ignore_mods", "Optionally filter the deletion to only messages sent by users who are not Moderators.")] bool ignoreMods = false,
+            [Option("match", "Optionally filter the deletion to only messages containing certain text.")] string match = "",
+            [Option("bots_only", "Optionally filter the deletion to only bots.")] bool botsOnly = false,
+            [Option("humans_only", "Optionally filter the deletion to only humans.")] bool humansOnly = false,
+            [Option("images_only", "Optionally filter the deletion to only messages containing images.")] bool imagesOnly = false,
```

Comment:
This should be renamed to `attachments_only` since that's all it does.
Not providing a suggestion because it would break several lines below.